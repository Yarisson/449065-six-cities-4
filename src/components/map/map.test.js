import React from 'react';
import Map from './map.jsx';
import renderer from 'react-test-renderer';

const hotels = [
  {img: `img/apartment-01.jpg`, price: `&euro;200`, width: `100%`, title: `Beautiful &amp; luxurious apartment at great location`, type: `Apartment`, coor: [52.390450945458, 4.04309666406198]},
  {img: `img/room.jpg`, price: `&euro;140`, width: `20%`, title: `Wood and stone place`, type: `Private room`, coor: [51.63072056168636, 4.05303667407681]},
  {img: `img/apartment-02.jpg`, price: `&euro;52`, width: `60%`, title: `Canal View Prinsengracht`, type: `Apartment`, coor: [52.00000000000000, 4.10000000000000]},
  {img: `img/apartment-03.jpg`, price: `&euro;85`, width: `80%`, title: `Nice, cozy, warm big bed apartment`, type: `Apartment`, coor: [52.10000000000000, 4.15000000000000]},
  {img: `img/room.jpg`, price: `&euro;150`, width: `40%`, title: `Wood and stone place`, type: `Private room`, coor: [52.15000000000000, 4.25000000000000]},
];

const coors = [52.38, 4.9];

const activeItem = [51.63072056168636, 4.05303667407681];

it(`Map component render`, () => {
  const tree = renderer
    .create(<Map
      hotels={hotels}
      coors={coors}
      activeItem={activeItem}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
