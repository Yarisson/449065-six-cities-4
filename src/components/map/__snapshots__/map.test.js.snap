// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map component render 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "849px",
      "width": "512px",
    }
  }
/>
`;
